winrm/client.go

```diff
@@ -18,10 +18,16 @@ type Client struct {
 	password  string
 	useHTTPS  bool
 	url       string
+	err       error
 	http      HttpPost
 	transport http.RoundTripper
 }
 
+type WinRMError struct {
```

Comment:
This is a left over I forgot to remove. All the errors are defined in `errors.go`, and only returned as pure Go `error`.